source/common/http/http2/codec_impl.h

```diff
@@ -43,6 +46,23 @@ namespace Envoy {
 namespace Http {
 namespace Http2 {
 
+enum ErrorType {
```

Comment:
```suggestion
enum class ErrorCode {
```

Using scoped enum will make it more readable throughout the code.